plugins/modules/openssl_pkcs12.py

```diff
@@ -402,6 +426,10 @@ def __init__(self, module, backend, iter_size_default=2048):
                 self.other_certificates = [
                     load_certificate(other_cert, backend=self.backend) for other_cert in self.other_certificates
                 ]
+        elif self.other_certificates_content:
+            self.other_certificates = [
+                load_certificate(None, content=to_bytes(other_cert), backend=self.backend) for other_cert in self.other_certificates_content
+            ]
```

Comment:
This is missing support for `other_certificates_parse_all`.

Reply:
I thought about this when writing the code, does it actually make sense to have for the content variant? Wouldn't be hard to add but probably not common?

Reply:
No idea... One possible use case would be loading the CA file (containing a list of PEMs) on the controller and running the module on the remote. If you can use that option here as well, you don't need to write the file to disk on the remote first.

Reply:
Added, since it doesn't take to much effort. Let's see what the tests say.